tf-build-tools/tf-maven-plugin/src/main/java/com/deliveredtechnologies/maven/terraform/TerraformGet.java

```diff
@@ -40,12 +42,12 @@
    * @param tfModules the common modules directory; if null, it's defaulted to src/main/.tfmodules
    */
   public TerraformGet(Log log, String tfModules) throws IOException {
-    this(log, TerraformUtils.getDefaultTfModulesDir());
+    this(log, StringUtils.isEmpty(tfModules) ? null : Paths.get(tfModules));
```

Comment:
This is a bug fix. Previously, when tfModules directory was supplied, it was ignored.